dashboard/routes/node/beneficiary.go

```diff
@@ -0,0 +1,100 @@
+package node
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/gin-gonic/gin"
+	. "github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/dashboard/auth"
+	"github.com/nknorg/nkn/dashboard/helpers"
+	"github.com/nknorg/nkn/util/config"
+	"github.com/nknorg/nkn/util/log"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+type SetBeneficiaryData struct {
+	BeneficiaryAddr string `form:"beneficiaryAddr" binding:"required"`
+}
+
+func BeneficiaryRouter(router *gin.RouterGroup) {
+	router.PUT("/node/beneficiary", auth.WalletAuth(), func(context *gin.Context) {
+		bodyData := helpers.DecryptData(context)
+
+		var data SetBeneficiaryData
+		err := json.Unmarshal([]byte(bodyData), &data)
+		if err != nil {
+			log.WebLog.Error(err)
+			context.AbortWithError(http.StatusBadRequest, err)
+			return
+		}
+
+		if data.BeneficiaryAddr != "" {
+			_, err = ToScriptHash(data.BeneficiaryAddr)
+			if err != nil {
+				log.WebLog.Errorf("parse BeneficiaryAddr error: %v", err)
+				context.AbortWithError(http.StatusBadRequest, err)
+				return
+			}
+		}
+
+		configFile := config.ConfigFile
```

Comment:
L43-L56 is redundant with util/config.go. It would be better if we create a function OpenConfigFile() in util/config.go and call it in both util/config.go and here.